2-GenSoftware.sh

```diff
@@ -88,7 +88,9 @@ mkdir -p ~/.config/autostart
 cp ./config_files/tilda.desktop ~/.config/autostart
 cp ./config_files/redshift-gtk.desktop ~/.config/autostart
 
-execute sudo apt-get install htop gparted expect -y
+execute sudo apt-get install htop cpufrequtils indicator-cpufreq gparted expect -y
+execute /usr/bin/cpufreq-info
```

Comment:
I'm still not sure what's the use of this command in the script, atleast with `execute` being used, which will just run the command and not show the output

Reply:
It's just running the app, so that the indicator shows up in the tray/panel

Reply:
`cpufreq-info` just gives the CPU info, for running the applet `indicator-cpufreq` has to be used
And that would actually stop the script, since it doesn't exit after running the command, needs to be killed